test/platform/test-platform-identify.ts

```diff
@@ -80,23 +82,24 @@ test('trigger-identify-with-undefined-verificationId-throws', async t => {
 test('trigger-identify-with-valid-state', async t => {
   const discoveryBuffer = Buffer.from('discovery buffer 123');
   const localDeviceId = 'local-device-id-123';
-  const validIdentifyHandler: smarthome.IntentFlow.IdentifyHandler = () => {
-    return {
-      requestId: 'request-id',
-      intent: smarthome.Intents.IDENTIFY,
-      payload: {
-        device: {
-          id: DEVICE_ID,
-          verificationId: localDeviceId,
-        },
-      },
-    };
-  };
   const app: smarthome.App = new smarthome.App(APP_VERSION);
-  app.onIdentify(validIdentifyHandler).onExecute(EXECUTE_HANDLER).listen();
+  const deviceManager = app.getDeviceManager();
+  const validIdentifyHandler = createIdentifyHandler(DEVICE_ID, localDeviceId);
+  const validExecuteHandler = createExecuteHandler(
+    createDeviceCommand(
+      smarthome.Constants.Protocol.UDP,
+      Buffer.from('execute-buffer'),
+      'execute-request-id',
+      DEVICE_ID,
+      12345
+    ),
+    deviceManager
+  );
+  app.onIdentify(validIdentifyHandler).onExecute(validExecuteHandler).listen();
```

Comment:
Is there an example of using an `invalidExecuteHandler`?

Reply:
Added invalid examples in `execute-handler-sends-wrong-buffer`.  Renamed this local variable to `executeHandler`